src/axom/core/utilities/BitUtilities.hpp

```diff
@@ -14,9 +14,21 @@
 #ifndef AXOM_BIT_UTILITIES_HPP
 #define AXOM_BIT_UTILITIES_HPP
 
-#include "axom/core/Macros.hpp"  // for AXOM_DEVICE_CODE
+#include "axom/config.hpp"
+#include "axom/core/Macros.hpp"
 #include "axom/core/Types.hpp"
 
```

Comment:
I added compiler intrinsics for our CPU-based compilers, which are used extensively in `slam::BitSet`
In some runs of the Delaunay triangulation, `popCount()` was taking >50% of the runtime.

Note: the underlying optimized assembly is only enabled when the code is built in an architecture-aware manner, e.g. via the `-march-native` flag. I'm planning to add a ticket for us to enable this, e.g. via a new config option

Reply:
See: #763